test/nofiles.test.js

```diff
@@ -0,0 +1,90 @@
+const stylelint = require('stylelint')
+const path = require('path')
+
+const code = `import styled from 'styled-components';
+const StyledTable = styled(StyledTableBase)\`
+color: red;
+\``
+const processor = path.join(__dirname, '../lib/index.js')
+const rules = {
+  indentation: 2
+}
+
+describe('no files', () => {
+  let codeFilename
+  let data
+
+  // NOTE beforeEach() runs _after_ the beforeAll() hooks of the describe() blocks
+  beforeEach(done => {
+    stylelint
+      .lint({
+        code,
+        codeFilename,
+        syntax: 'scss',
+        config: {
+          processors: [processor],
+          rules
+        }
+      })
+      .then(result => {
+        data = result
+        done()
+      })
+      .catch(err => {
+        // eslint-disable-next-line
+        console.log(err)
+        data = err
+        done()
+      })
+  })
+
+  describe('no codeFilename', () => {
+    beforeAll(() => {})
```

Comment:
This is redundant right? We can just delete it. Unless you wanna assign `codeFileName = undefined;` specifically, but tests in Jest are guaranteed to run in order right? So I think I'm okay with both, deleting or explicitly defining it to be `undefined`

Reply:
Yes, I defined it explicitly.